app/controllers/shopping_lists_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ShoppingListsController < ApplicationController
+  def index
+    @users = User.all.includes(:foods)
+    @user = User.includes(:foods).find(params[:user_id])
+    @foods = Food.where.not(user_id: current_user.id).includes(:user)
+  end
+end
```

Comment:
Your shopping list is not displaying the correct data. I have recorded [this ](https://www.loom.com/share/c15d089a799e485bab9efb56c61429dd?sid=15602e16-d36e-46d1-a72b-5eac8750963a)video where i explained what the issue is

Reply:
well resolved! ✔️